service_list.yaml

```diff
@@ -81,6 +81,17 @@ wideEncodeIdx: [8075, true, 20.]
 wideFrame: [8076, true, 20.]
 modelV2: [8077, true, 20., 20]
 
+sensorAccelerometer1: [8078, true, 20, 1]
+sensorAccelerometer2: [8079 , true, 20, 1]
```

Comment:
these frequencies are wrong. do we want them in qlogs?

Reply:
This was just a proposal. Need to look up the frequencies and we need to decide what we want to keep for each sensor.

Reply:
think we want to do this in another PR anyway